test/hotspot/jtreg/runtime/verifier/OverriderMsg.java

```diff
@@ -96,12 +96,17 @@ public static void dump_Overrider () throws Exception {
                         .withSuperclass(ClassDesc.of("HasFinal"))
 
                         .withMethod(INIT_NAME, MTD_void, ACC_PUBLIC,
+                                mb -> mb.withCode(
```

Comment:
If you only add code to a method, you can use the `withMethodBody` shorthand instead of `withMethod` plus `withCode`

Reply:
Done